### ISSUE TITLE
Downloaded files are not being renamed from "Partial" to "Downloaded" directory

### ISSUE THREAD

**Issue by ppetr** [RetroShare/RetroShare#2722]
I'm not sure what might be specific for my set-up. But files are not renamed as expected. They remain in the "partial" directory named just by their hash, even though they are shown as completed  by RetroShare. Is there a way to figure out why the renaming is failing?

**Comment by zeners** [comment]
maybe a duplicate of #2433?

**Comment by ppetr** [comment]
Yes, that's the same problem, thank you. I didn't find it before. Feel free to dedup against that issue.

**Comment by defnax** [comment]
@csoler close
